backend/src/main/java/com/karankumar/bookproject/backend/model/Book.java

```diff
@@ -244,6 +246,17 @@ public BookBuilder edition(Integer edition) {
         }
     }
 
+    public Publisher addBookToPublisher(@NonNull Book book, @NonNull Publisher publisher) {
```

Comment:
Could you rename this to `addPublisher()`? We can use `this` to refer to the current book and add the incoming `Publisher` parameter to the set (the implementation should b similar to `addPredefinedShelf()`

Reply:
@knjk04 we are adding book to publisher in PublisherService,java class's function. To remove that we should add that method in Publisher.java no??? .

And this to refer to the current book and add the incoming Publisher   ----------->. for this we will add publisher to a book, isn't it??

Reply:
> we are adding book to publisher in PublisherService,java class's function. To remove that we should add that method in Publisher.java no??? .

Ah, you're right. Could you move this to `Publisher.java`?

> And this to refer to the current book and add the incoming Publisher ----------->. for this we will add publisher to a book, isn't it??

Yes, so `this` would refer the publisher and we can have a `book` argument

Reply:
@knjk04 please check now.